Formula/gdcm.rb

```diff
@@ -1,36 +1,26 @@
 class Gdcm < Formula
   desc "Grassroots DICOM library and utilities for medical files"
   homepage "https://sourceforge.net/projects/gdcm/"
-  url "https://downloads.sourceforge.net/project/gdcm/gdcm%202.x/GDCM%202.8.7/gdcm-2.8.7.tar.gz"
-  sha256 "7a08baa93e90bce17d9999d59b95876808801a287812348e27a23decb1ebc58c"
+  url "https://downloads.sourceforge.net/project/gdcm/gdcm%202.x/GDCM%202.8.9/gdcm-2.8.9.tar.gz"
+  sha256 "a2da88b7b3cbf9e76a9df3e89d06d057cca9ce54fc62fb059e04f47bf056b727"
+  revision 1
```

Comment:
This should only be added when something is changed in homebrew without an upstream change.